templates/repo/issue/view_content/update_branch_by_merge.tmpl

```diff
@@ -19,8 +19,8 @@
 						<div class="ui dropdown icon button no-text">
 							{{svg "octicon-triangle-down" 14 "dropdown icon"}}
 							<div class="menu">
-								<div class="item active selected" data-do="{{.Link}}/update">{{$.locale.Tr "repo.pulls.update_branch"}}</div>
-								<div class="item" data-do="{{.Link}}/update?style=rebase">{{$.locale.Tr "repo.pulls.update_branch_rebase"}}</div>
+								<button class="item active selected" data-do="{{.Link}}/update">{{$.locale.Tr "repo.pulls.update_branch"}}</button>
+								<button class="item" data-do="{{.Link}}/update?style=rebase">{{$.locale.Tr "repo.pulls.update_branch_rebase"}}</button>
```

Comment:
It could be `<a>` instead of `<button>`. Then `aria.js` could click them.

Reply:
Wait, I'm confused now.
Isn't that exactly the case described in https://github.com/go-gitea/gitea/pull/23337#discussion_r1130842438?

Reply:
No, items in dropdown are different. `aria.js` only  handles `a.item` or `.item.js-aria-clickable` CSS selectors in dropdown.